assets/js/reviewerAction.js

```diff
@@ -0,0 +1,30 @@
+$(function () {
```

Comment:
The filename needs to be more descriptive, i.e. it needs to show that it is for reporting purposes only. How about `reviewerAnalytics`?